nano/lib/timer.hpp

```diff
@@ -0,0 +1,250 @@
+#pragma once
+
+namespace nano
+{
+enum timer_state
```

Comment:
This should use the modern **enum class**

Reply:
Yep, thanks